pootle/apps/pootle_fs/getters.py

```diff
@@ -59,3 +64,16 @@ def fs_url_validator_getter(**kwargs_):
 @getter(fs_translation_mapping_validator)
 def fs_translation_mapping_validator_getter(**kwargs_):
     return TranslationMappingValidator
+
+
+@getter([config_should_not_be_set, config_should_not_be_appended], sender=Project)
+def fs_url_config_validator(**kwargs):
+    if kwargs["key"] == "pootle_fs.fs_url":
+        fs_type = kwargs["instance"].config.get("pootle_fs.fs_type")
+        if not fs_type:
+            raise ValidationError("You cannot set fs url until fs type is set")
+        plugin_class = fs_plugins.gather(Project).get(fs_type)
+        if plugin_class:
```

Comment:
Shouldn't some error be thrown in case no plugin can be fetch?

Reply:
see above comment